### PR TITLE
Replce 'rake routes' task with 'rails rotues'

### LINKED ISSUE #9661: `rake routes` task is deprecated
**Describe the bug**
A clear description of what the bug is.
The `rake routes` task is replaced by `rails routes` task. ([Related PR](https://github.com/rails/rails/pull/23225))

**To Reproduce**
Steps to reproduce the behavior, for example:
1. Enable this plugin 'rails'
2. Run command 'rr'
3. See the following error
```
rake aborted!
Don't know how to build task 'routes' (See the list of available tasks with `rake --tasks`)
```

**Expected behavior**
The task should show the Rails routes

### PR DESCRIPTION
**Pull request by tannakartikey**
Fixes https://github.com/ohmyzsh/ohmyzsh/issues/9661

## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

- Replaced `rake routes` task with `rails routes` (Fixes #9661)

### CODE REVIEW

File: plugins/rails/rails.plugin.zsh [hunk 1]
```diff
@@ -65,7 +66,6 @@ alias rdmtc='rake db:migrate db:test:clone'
 alias rdsl='rake db:schema:load'
 alias rlc='rake log:clear'
 alias rn='rake notes'
-alias rr='rake routes'
 alias rrg='rake routes | grep'
```
Comment (Volosh1n) [inline plugins/rails/rails.plugin.zsh:69]: I think this should be changed too

Comment (tannakartikey) [reply plugins/rails/rails.plugin.zsh]: @Volosh1n yes, I missed that. Added a new commit.

**Review by robbyrussell — APPROVED** [review]
Looks good

**Comment by robbyrussell** [comment]
Hey there -- we really appreciate the contribution!

**Pull request merged**